operations/mimir-mixin/alerts/alerts.libsonnet

```diff
@@ -513,6 +531,7 @@ local utils = import 'mixin-utils/utils.libsonnet';
             kube_statefulset_status_update_revision: groupStatefulSetByRolloutGroup('kube_statefulset_status_update_revision'),
             kube_statefulset_replicas: groupStatefulSetByRolloutGroup('kube_statefulset_replicas'),
             kube_statefulset_status_replicas_updated: groupStatefulSetByRolloutGroup('kube_statefulset_status_replicas_updated'),
+            range_interval: $.alertRangeInterval(15) + ':' + $.alertRangeInterval(1),
```

Comment:
we talked privately about keeping anything <=10m hardcoded. Should we keep these hardcoded too? At least the `15m` part. The same applies below

Reply:
fixed